### PR TITLE
Added template for VM with Managed Disks

### PR DESCRIPTION
**Pull request by jomit**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@jomit__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jovagh). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-vm-manageddisks/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        "description": "Password for the Virtual Machine."
+      }
+    },
+    "dnsLabelPrefix": {
+      "type": "string",
+      "metadata": {
+        "description": "Unique DNS Name for the Virtual Machine."
+      }
+    },
+    "vmSize": {
```
Comment (bmoore-msft) [inline 101-vm-manageddisks/azuredeploy.json]: move to variable and use a size available in all clouds (e.g. Dv2)

File: 101-vm-manageddisks/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+      "type": "string",
+      "metadata": {
+        "description": "Size of VM"
+      }
+    }
+  },
+  "variables": {
+    "addressPrefix": "10.0.0.0/16",
```
Comment (bmoore-msft) [inline 101-vm-manageddisks/azuredeploy.json]: most of these variables are only used once, just use a literal property value for those

File: 101-vm-manageddisks/azuredeploy.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+    "imageSKU": "14.04.2-LTS",
+    "imageVersion": "latest",
+    "publicIPAddressName": "myPublicIP",
+    "publicIPAddressType": "Dynamic",
+    "storageAccountType": "Standard_LRS",
+    "virtualNetworkName": "myVNET",
+    "vmName": "myVM",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "nicName": "myNIC",
+    "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",
```
Comment (bmoore-msft) [inline 101-vm-manageddisks/azuredeploy.json]: use resourceId() and then can remove vnetID above

File: 101-vm-manageddisks/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    "apiVersion": "2015-06-15",
+    "computeResouresApiVersion": "2016-04-30-preview",
```
Comment (bmoore-msft) [inline 101-vm-manageddisks/azuredeploy.json]: use literal values for apiVersion so schema validation can be done

**Comment by bmoore-msft** [comment]
@gatneil - have you updated any of the 100-level samples to include this scenario?

**Comment by bmoore-msft** [comment]
now a duplicate of other md samples

**Pull request closed without merging**